webapp/packages/plugin-connections/src/DatabaseAuthDialog/DatabaseAuthDialog.tsx

```diff
@@ -94,7 +98,7 @@ export const DatabaseAuthDialog: DialogComponent<Payload> = observer(function Da
             <ConnectionAuthenticationForm
```

Comment:
please add to `ConnectionAuthenticationForm` property: `hideFeatures` which is a list of features that's shuldn't be displayed